CMakeLists.txt

```diff
@@ -0,0 +1,328 @@
+#   realsense, timewarp_gl
+#
+# In addition to the standard CMake options, the following arguments can be used:
+#
+# BUILD_GROUP [ALL|NATIVE|MONADO|CI|TESTS|NONE] : which subgroup of plugins are to be built
+#     ALL : all plugins, this is the default
+#     NATIVE : audio_pipeline, debugview, gldemo, gtsam_integrator, ground_truth_slam,
+#              kimera_vio, offline_imu, offline_cam, offload_data, pose_prediction, timewarp_gl
+#     MONADO : audio_pipeline, gtsam_integrator, kimera_vio, monado, offline_imu, offline_cam,
+#              openxr_app, pose_prediction, timewarp_gl
+#     CI : audio_pipeline, gldemo, ground_truth_slam, gtsam_integrator, kimera_vio,
+#          offline_imu, offline_cam, pose_prediction,  timewarp_gl
+#     NONE : no group is built, individual plugins can be selected
+#
+# To build specific plugins use the following format:
+#   USE_<PLUGIN>=ON
+# where <PLUGIN> is the name of the plugin in upper case.
+#
+########################################################################################
+
+cmake_minimum_required(VERSION 3.16)
+set(CMAKE_VERBOSE_MAKEFILE True)
+if (POLICY CMP0048)
+    cmake_policy(SET CMP0048 NEW)
+endif (POLICY CMP0048)
+
+if (POLICY CMP0135)
+    cmake_policy(SET CMP0135 NEW)
+endif()
+
+if (POLICY CMP0077)
+    cmake_policy(SET CMP0077 NEW)
+endif()
+
+set(BUILDING_ILLIXR Yes)
+
+set(PACKAGE_NAME "ILLIXR")
+set(PACKAGE_STRING "ILLIXR 1.0.0")
+set(PACKAGE_VERSION_MAJOR "1")
+set(PACKAGE_VERSION_MINOR "0")
+set(PACKAGE_VERSION_PATCH "0")
+set(PACKAGE_VERSION_LABEL "")
+
+if(UNIX)
+    execute_process(COMMAND bash -c "awk -F= '/^ID=/{print $2}' /etc/os-release | tr -d '\n' | tr -d '\"'" OUTPUT_VARIABLE OS_FLAVOR)
+    message("Found OS: ${OS_FLAVOR}")
+    string(TOLOWER "${OS_FLAVOR}" OS_LOWER)
+    if(OS_LOWER STREQUAL "centos")
+        set(HAVE_CENTOS Yes)
+    else()
+        set(HAVE_CENTOS No)
+    endif()
+endif()
+
+if(PACKAGE_VERSION_LABEL)
+    set(PACKAGE_VERSION "${PACKAGE_VERSION_MAJOR}.${PACKAGE_VERSION_MINOR}.${PACKAGE_VERSION_PATCH}-${PACKAGE_VERSION_LABEL}")
+else()
+    set(PACKAGE_VERSION "${PACKAGE_VERSION_MAJOR}.${PACKAGE_VERSION_MINOR}.${PACKAGE_VERSION_PATCH}")
+endif()
+
+include(CMakeDependentOption)
+include(ExternalProject)
+
+project(ILLIXR VERSION 1.0.0)
+set(CMAKE_CXX_STANDARD 17)
+set(CMAKE_CXX_STANDARD_REQUIRED ON)
+add_definitions(-DGLSL_VERSION="330")
+set(ILLIXR_BUILD_SUFFIX "")
+if(CMAKE_BUILD_TYPE STREQUAL "")
+    message("REL")
+    set(CMAKE_BUILD_TYPE "Release")
+endif()
+
+string(TOLOWER "${CMAKE_BUILD_TYPE}" lower_type)
+if(lower_type MATCHES "debug")
+    set(ILLIXR_BUILD_SUFFIX ".dgb")
+elseif(lower_type MATCHES "release")
+    set(ILLIXR_BUILD_SUFFIX ".opt")
+elseif(lower_type MATCHES "relwithdebinfo")
+    set(ILLIXR_BUILD_SUFFIX ".optdbg")
+endif()
+set(CMAKE_CXX_FLAGS_DEBUG "-ggdb -Wall -Wextra -fPIC")
+set(CMAKE_CXX_FLAGE_RELEASE "-O3 -DNDEBUG -Wall -Wextra -fPIC")
```

Comment:
Typo here

Reply:
Can you be more specific?

Reply:
I believe `CMAKE_CXX_FLAGE_RELEASE` should be `CMAKE_CXX_FLAGS_RELEASE`. @wermos first spotted this in #381

Reply:
Fixed, thanks.